scripts/migrate-black.py

```diff
@@ -51,7 +51,7 @@ def blackify(base_branch: str, black_command: str, logger: logging.Logger) -> in
                 "diff",
                 "--binary",
                 "--find-copies",
-                "%s-black..%s-black" % (last_commit, commit),
+                "{}-black..{}-black".format(last_commit, commit),
```

Comment:
```suggestion
                f"{last_commit}-black..{commit}-black",
```

Reply:
If you have any complains about how pyupgarde works please discuss that with pypgrade maintainer(s).
IMO cherry-picking "what I like/what I don't" does not make to much sense.

It is great value that it is GoodEnough™️ tool allowing to clean some code cleanups between major python versions.

You know that `black` code can be filtered to cut off some legacy tails.
If you don't like this PR just close it .. really no one will be hurt.

FYI: in only few weeks 3.8 will be EOSed and with that version can be dropped much more.

Reply:
Hi, sorry to hear you're not a fan of my suggestion!

> IMO cherry-picking "what I like/what I don't" does not make to much sense.

Not sure I agree there -- in my opinion, any proposed change should be considered on its own merits, regardless of whether the change has come about as a result of an autofix or codemod. But I'm not a Black maintainer, so the decision on how best to review this isn't up to me 😊

> If you don't like this PR just close it .. really no one will be hurt.

Well, again, I'm not a maintainer, so I don't personally in fact have this power

Reply:
> FYI: in only few weeks 3.8 will be EOSed

Python 3.8 is scheduled to go End Of Life in October 2024, FWIW, so we have at least six months to go yet